app/controllers/application_controller.rb

```diff
@@ -1,4 +1,25 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  helper_method :current_user, :require_login
```

Comment:
Helpers are for the `views` [0] and I don't see usage of these methods in the views

[0] https://api.rubyonrails.org/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper_method